src/backend/ethereum/index.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path';
+
+import * as fs from 'fs-extra';
+
+import { ETHEREUM_PROJECT_GENESIS } from '../../data';
+
+/**
+ * Creates a genesis.json file.
+ *
+ * @param {string} folder - The folder to put genesis.json in.
+ */
+export function createGenesis(folder: string) {
+  const file = path.join(folder, 'genesis.json');
+  if (fs.existsSync(file)) {
+    throw new Error('genesis file already exists');
+  }
+  fs.ensureDirSync(folder);
```

Comment:
I'm not completely sure but if a file with the folder's path already exists, would this throw an exception? Do we want to catch it?

Reply:
Yes it's gonna throw an error, just like the above. I think we should just let functions throw error and then catch them at wherever needed.